site/en/docs/extensions/mv3/service_workers/service-worker-lifecycle/index.md

```diff
@@ -41,17 +41,15 @@ When a user profile starts, the [`chrome.runtime.onStartup`](/docs/extensions/re
 
 ## Idle and shutdown {: #idle-shutdown }
 
-Extension service workers are dormant unless an event fires. If the service worker has a handler for the fired event, it wakes up, executes it, then goes back to being idle. If the service worker is idle for at least 30 seconds it shuts down. You should design your service worker to be resilient against unexpected termination, since not all activity keeps the service worker alive. For example, this can interrupt a service worker [`fetch()`](https://developer.mozilla.org/docs/Web/API/fetch) call if the response takes more than 30 seconds to arrive.
+Normally, Chrome terminates a service worker after one of the following conditions is met:
 
-Any new events and calls to extension APIs reset the idle timer the moment they're fired. For example, when a service worker receives a [`chrome.bookmarks.onCreated`](/docs/extensions/reference/bookmarks/#event-onCreated) event, the 30 second timeout is reset. The same is true when calling an extension API such as [`chrome.storage.local.get()`](/docs/extensions/reference/storage/#property-local) which also resets the 30 second timeout. As with web service workers, extension service workers have no shutdown or deactivation events.
+-   After 30 seconds of inactivity. Receiving an event or calling an extension API resets this timer.
+-   When a single request, such as an event or API call, takes longer than 5 minutes to process.
+-   When a [fetch()](https://developer.mozilla.org/docs/Web/API/fetch) response takes more than 30 seconds to arrive.
 
-Before Chrome 110, only running event handlers caused the idle time to reset. Any events that were queued, but for which a handler had not been called would not cause a reset. Also, extension service workers had a maximum lifetime of five minutes before Chrome shut them down. These behaviors caused service workers to shut down at unexpected times.
+Events and calls to extension APIs reset these timers, and if the service worker has gone dormant, they will revive them. Nevertheless, you should design your service worker to be resilient against unexpected termination.
 
-Though Chrome 110 changed this you should not keep your service worker alive indefinitely, though you can. (We don't consider this a good programming practice.) You should test your extensions to ensure that they're not doing this unintentionally.
-
-Though it's difficult to catch in a code review, we suggest you guard against it as best you can.
-
-For all of these reasons, you need to guard against unexpected termination of the service worker.
+You should not keep your service worker alive indefinitely, though you can. (We don't consider this a good programming practice.) You should test your extensions to ensure that they're not doing this unintentionally. Though it's difficult to catch in a code review, we suggest you guard against it as best you can.
```

Comment:
Suggestion: I think we can simplify this paragraph to the following:

"To optimize the resource consumption of your extension, avoid keeping your service worker alive indefinitely if possible. Test your extensions to ensure that you're not doing this unintentionally."

Reply:
Done.